jupyter_client/kernelspec.py

```diff
@@ -205,14 +173,10 @@ def install_native_kernel_spec(self, user=False):
         kernel registry. If the process does not have appropriate permissions,
         an :exc:`OSError` will be raised.
         """
-        path = self._get_destination_dir(NATIVE_KERNEL_NAME, user=user)
-        os.makedirs(path, mode=0o755)
-        with open(pjoin(path, 'kernel.json'), 'w') as f:
-            json.dump(self._native_kernel_dict, f, indent=1)
-        copy_from = self._native_kernel_resource_dir
-        for file in os.listdir(copy_from):
-            shutil.copy(pjoin(copy_from, file), path)
-        return path
+        from ipython_kernel.kernelspec import write_kernel_spec
```

Comment:
I'm not quite sure. I originally thought it would be, but if Jupyter projects are going to have the `ensure_ipython_present`, maybe not? Maybe there should be ipython_kernel.kernspec.install_self(), in which case I can deprecate this method.